streamr-docker-dev/bin.sh

```diff
@@ -93,6 +93,14 @@ bind_ip() {
     COMMANDS_TO_RUN+=("sudo ifconfig lo0 alias 10.200.10.1/24")
 }
 
+pull(){
+    # Pull latest images define on docker compose
+    # --all for all services, Service name for specific service
+    check_services_from_arguments
```

Comment:
I think this check will always pass since we are setting `ALL=1` before calling `pull`. Should we just remove this line?

Reply:
Oh, misread `check_services_from_arguments`. There is an AND not OR 🤦‍♂